src/widgets/JSONForm/widgets/Textarea.js

```diff
@@ -28,6 +37,7 @@ export const Textarea = ({ autofocus, disabled, placeholder, value, onChange, nu
       autoFocus={autofocus}
       multiline={true}
       numberOfLines={numberOfLines}
+      onEndEditing={handleBlur}
```

Comment:
this method returns the value of the text input, the `onBlur` doesn't with multiline text inputs